docs/CHANGELOG.rst

```diff
@@ -24,13 +19,16 @@ Changed
 * Update differential expression (DE) processors
 * Update ``generate_diffexpr_cuffdiff`` django-admin command
 * Save gene_id source to ``output.source`` for DE, expression and related objects
+* Update sample descriptor schema
 
 Added
 -----
 * Process definition documentation style and text improvements
+* Add tests to ensure generators produce the same results
```

Comment:
Are test supposed to be in this commit? I can't find them. I ran the generators manually and they seem to work.